pyproject.toml

```
@@ -29,6 +29,9 @@ dev = [
 
 [project.entry-points."console_scripts"]
 hub_predtimechart = "hub_predtimechart.app.generate_json_files:main"
+ptc_generate_json_files = "hub_predtimechart.app.generate_json_files:main"
+ptc_generate_FluSight_targets = "hub_predtimechart.app.generate_target_json_files_FluSight:main"
```

It works either way, but it seems awkward to have camel case in a command line function

```suggestion
ptc_generate_FluSight_targets = "hub_predtimechart.app.generate_target_json_files_flusight:main"
```

---

I agree with this assessment, but the suggestion would cause the app to break?

```suggestion
ptc_generate_flusight_targets = "hub_predtimechart.app.generate_target_json_files_FluSight:main"
```

---

Oh, my bad--your version is right; I meant un-camel-case the first item, not the second ⛔ 🐫